keps/sig-instrumentation/20190131-event-series.md

```diff
@@ -16,7 +16,7 @@ approvers:
 editor: TBD
 creation-date: 2019-01-31
 last-updated: 2019-01-31
-status: implementable
+status: implemented
```

Comment:
This isn't fully implemented - we're working on that.

Reply:
Please revert

Reply:
What's the status of this now? before I redo this PR